pkg/arm/v11/resources.go

```diff
@@ -243,15 +284,41 @@ func (g *simpleGenerator) ilbAPIServer() *network.LoadBalancer {
 					},
 					Name: to.StringPtr(armconst.LbAPIServerLoadBalancingRuleName),
 				},
+				{
+					LoadBalancingRulePropertiesFormat: &network.LoadBalancingRulePropertiesFormat{
+						FrontendIPConfiguration: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								armconst.IlbAPIServerName,
+							) + "/frontendIPConfigurations/" + armconst.IlbAPIServerFrontendConfigurationName),
+						},
+						BackendAddressPool: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								armconst.IlbAPIServerName,
+							) + "/backendAddressPools/" + armconst.LbAPIServerBackendPoolName),
+						},
+						Protocol:             network.TransportProtocolTCP,
+						LoadDistribution:     network.Default,
+						FrontendPort:         to.Int32Ptr(22),
+						BackendPort:          to.Int32Ptr(22),
+						IdleTimeoutInMinutes: to.Int32Ptr(15),
+						EnableFloatingIP:     to.BoolPtr(false),
+					},
+					Name: to.StringPtr(armconst.LbSSHLoadBalancingRuleName),
+				},
 			},
 			Probes: &[]network.Probe{
 				{
 					ProbePropertiesFormat: &network.ProbePropertiesFormat{
-						Protocol:          network.ProbeProtocolHTTPS,
+						Protocol:          network.ProbeProtocolTCP,
```

Comment:
does that want to be reverted?

Reply:
(and the /healthz deletion below too)

Reply:
yes, but at the time of pushing i still needed it. the fix should be rolled out by now, i'll test

Reply:
I chatted to Julien this morning in the scrum and he said the azure fix is not totally rolled out yet
but when it is, we need to revert it (2 liner). I'd suggest merge as-is and after a couple of days revert it.